ReactNativeTemplate/index.ios.js

```diff
@@ -48,7 +48,7 @@ var App = React.createClass({
 
     componentDidMount: function() {
         var that = this;
-        oauth.authenticate(
+        oauth.getAuthCredentials(
```

Comment:
`authenticate` on `iOS` always brings up the login screen. We need `getAuthCredentials`, which brings up the login screen only if required.